src/InformaticsGateway/Common/TypeExtensions.cs

```diff
@@ -55,9 +55,9 @@ public static Type GetType(this Type interfaceType, string typeString)
                       typeString,
                       (name) =>
                       {
-                          var assembly = AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(z => !string.IsNullOrWhiteSpace(z.FullName) && z.FullName.StartsWith(name.FullName));
+                          var assembly = AppDomain.CurrentDomain.GetAssemblies().ToList().Find(z => !string.IsNullOrWhiteSpace(z.FullName) && z.FullName.StartsWith(name.FullName));
```

Comment:
Doesn't `.ToList().Find(...)` take extra execution?

Reply:
This was to stop a warning, but Im going to change it back.